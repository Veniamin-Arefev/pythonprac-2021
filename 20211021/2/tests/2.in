:sin x y k z sin(x+y+k+z)
sin 1 2 3 4
:decorate s "--<<{}>>--".format(s)
decorate "ЖЖЖ"
quit всё