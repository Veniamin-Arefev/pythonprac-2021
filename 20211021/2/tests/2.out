-0.5440211108893698
--<<ЖЖЖ>>--
5:3
